bpf/bpf_lxc.c

```diff
@@ -56,7 +56,8 @@
  */
 #if !defined(ENABLE_HOST_SERVICES_FULL) || \
     defined(ENABLE_SOCKET_LB_HOST_ONLY) || \
-    defined(ENABLE_L7_LB)
+    defined(ENABLE_L7_LB)               || \
+    defined(ENABLE_SCTP)
```

Comment:
What's the rationale for this change? AFAIU, since we can't change the ports, neither socket-level LB nor per-packet LB will be supported anyway.

Reply:
Hm, I thought that per-packet LB was required to do lookups for service IPs. I recall having weird issues without this enabled (something where we would continue to send packets with the Service IP instead of the translated Pod IP). I can try again with this enabled and see what happens.

Reply:
Oh, right! Socket-level LB is the one case of DNAT where things will work fine despite not being able to change the checksum, because we DNAT before the packets are even generated. So this change makes sense. Maybe the documentation should clarify this point though.